config/config_schema.json

```diff
@@ -0,0 +1,74 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "DRS Compliance Suite Config",
+  "description": "DRS Compliance Suite Config v1.2",
+  "type": "object",
+  "$ref": "config_schema.json#/definitions/config_schema",
+  "definitions": {
+    "config_schema": {
+      "type": "object",
+      "properties": {
+        "default_auth_info": {
```

Comment:
Add drs_objects as required and maybe minimum elements as "3" (Additionally, Is it possible in JSON schema to ensure that these 3 objects are not duplicates?) This way we can mandate that the user is testing their DRS implementation with at least 3 different DRS ids